pages/clubs.jsx

```diff
@@ -139,10 +139,10 @@ var Quote = React.createClass({
     return(
       <div className="row">
         <div className="col-sm-offset-2 col-sm-8 col-md-offset-2 col-md-8 col-lg-offset-2 col-lg-8">
-          <Blockquote className="primary-quote" author="Mikko K, Helsinki, Finland"
```

Comment:
Yea, I realized the majority of the CSS code in [`home.less`](https://github.com/mozilla/teach.webmaker.org/blob/develop/less/pages/home.less) is outdated/no longer in use so I cleaned that up.  Same with `primary-quote`.